data/nlu/responses/responses_schlafen.yml

```
@@ -9,27 +9,28 @@ responses:
         - type: slot
           name: volljaehrig
           value: True
-      text: "Diese Stellen helfen dir, eine langfristige Unterkunft für Erwachsene zu finden."
+      text: "Diese Stellen helfen dir, eine langfristige Unterkunft für Erwachsene zu finden: \n - Amt für Existenzsicherung und soziale Integration - Sozialamt
```

```suggestion
      text: |+ 
          Diese Stellen helfen dir, eine langfristige Unterkunft für Erwachsene zu finden: 
            - Amt für Existenzsicherung und soziale Integration
            - Sozialamt
```
Finde die Schreibweise besser?